File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_evalmany.java
// EXPECTED: 25
// Reference emit for EVALMANY — multi-WHEN EVALUATE.
public final class EvalmanyRef {
  public static void main(String[] args) {
    int code = 2;
    int fee = switch (code) {
      case 1 -> 10;
      case 2 -> 25;
      case 3 -> 40;
      default -> 99;
    };
    System.out.println(fee);
  }
}

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_evalmany.cs
// EXPECTED: 25
// Reference emit for EVALMANY — multi-WHEN EVALUATE.
using System;
public static class EvalmanyRef {
  public static void Main() {
    int code = 2;
    int fee = code switch {
      1 => 10,
      2 => 25,
      3 => 40,
      _ => 99
    };
    Console.WriteLine(fee);
  }
}
